server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
```

Comment:
This is not required. We have [a dedicated backend service](https://github.com/ably/asset-tracking-backend-demo) for this suite of demos.

Therefore, there is no reason why this Web demo should not be created as a static site.

Reply:
@QuintinWillison the library loads the static files generated in the `dist` folder after builing [asset-tracking-js](https://github.com/ably/ably-asset-tracking-js). I updated the readme to include that information. I'm not exactly sure if that covers what you mean. If not, do you mind explaining further?